mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -141,6 +141,19 @@ func stop() error {
 			return err
 		}
 
+		// check if deployments exists in manifest folder
+		if _, err := os.Stat(filepath.Join(utils.MesheryFolder, utils.ManifestsFolder, utils.MesheryDeployment)); os.IsNotExist(err) {
```

Comment:
@suhaniii is this logic considering the specific version of Meshery that the current-context is using?

Reply:
@piyushsingariya @metonymic-smokey @navendu-pottekkat are we on a path to having a structured set of folder-versioned manifests?

Reply:
@leecalcote we have a design spec for air gapped deployments. We could add considerations for it there, I'll work towards adding them.

Reply:
@leecalcote  It is just checking if `MesheryDeployment` is present during `system stop`. 
What else do I need to consider?

Reply:
Thank you, @metonymic-smokey. Wonderful.

Reply:
@suhaniii, I think I'm jumping the gun with my remarks and that your update here is inline with the current architecture. Good stuff. ✔️